packages/webapp/src/containers/Finances/styles.module.scss

```diff
@@ -51,8 +51,27 @@
 }
 
 .filterBar {
+  width: 100%;
   display: flex;
+  justify-content: space-between;
   gap: 8px;
   height: 48px;
   margin-bottom: 8px;
 }
+
+.filterBarButtons {
+  display: flex;
+  gap: 8px;
+}
+
+.dateRangeSelector {
+  max-width: 360px;
```

Comment:
This is actually larger than what is shown in Figma, but I was worried about the text not fitting.